docs/source/quickstart.rst

```diff
@@ -0,0 +1,47 @@
+Quickstart
+==========
+
+This guide will show you how to get started using XrViz
+to explore your xarray data.
+
+Installation
+------------
+
+If you are using `Anaconda`_ or Miniconda, install Intake
+with the following commands::
+
+    conda install -c conda-forge xrviz
+    pip install git+https://github.com/intake/xrviz --no-deps
+
+If you are using virtualenv/pip, run the following command::
+
+    pip install git+https://github.com/intake/xrviz
+
+Note that this will install with the mininum of optional requirements.
+If you want a more complete install, use conda instead.
+
+.. _Anaconda: https://www.anaconda.com/download/
+
+
+Example
```

Comment:
Pointing out the example is fine, but you totally don't say how to display any other data! You have not given the most basic way people to use the software...